website/config/_default/hugo.toml

```diff
@@ -0,0 +1,89 @@
+title = "GDG"
+baseurl = "https://software.es.net/gdg/"
+canonifyurls = false
+
+relativeURLs = false
+
+disableAliases = true
+disableHugoGeneratorInject = true
+# disableKinds = ["taxonomy", "term"]
+enableEmoji = true
+enableGitInfo = false
+enableRobotsTXT = true
+languageCode = "en-US"
+paginate = 10
+rssLimit = 10
+summarylength = 20 # 70 (default)
+
+# Multilingual
+defaultContentLanguage = "en"
+disableLanguages = ["de", "nl"]
+defaultContentLanguageInSubdir = false
+
+copyRight = "Copyright (c) 2020-2024 Hyas"
```

Comment:
Is this asserting that GDG's copyright is Hyas? I think this is for the DOKS system, not the application?

Reply:
Just default template, but removed.